R/cal-estimate-utils.R

```diff
@@ -166,7 +264,13 @@ as_cal_object <- function(estimate,
     cli::cli_abort("Cannot translate {.arg levels} to a class.")
   }
 
-  str_truth <- as_name(enquo(truth))
+  # TODO remove this when everything is updated/refactored
+  refactored <- c("cal_estimate_multinomial")
```

Comment:
This is a crude way of handling the differences in the value types as we continue to refactor.